Permanent/Modules/Onboarding/Views/OnboardingCreateFirstArchiveView.swift

```diff
@@ -7,37 +7,74 @@
 import SwiftUI
 
 struct OnboardingCreateFirstArchiveView: View {
+    @State var archiveType: String = "Personal"
+    
     var backButton: (() -> Void)
     var nextButton: (() -> Void)
     
     var body: some View {
-        ZStack(alignment: .bottom) {
-            VStack {
+        if Constants.Design.isPhone {
```

Comment:
I think that it would be easier to read if you would move the body into two separate views like this:

```
var body: some View {
if Constants.Design.isPhone {
  iphoneBody
else
  ipadBody
}

var iphoneBody: some View {
////content
}

var iPadBody: some View {
//content
}
```